### ISSUE TITLE
Very little result

### ISSUE THREAD

**Issue by Li-An** [johngodley/search-regex#13]
### Summary

I have a lot of linked thumbnails in my posts with **alignnone** style and I want to change them into **aligncenter**.
Searchregex before 2.0 gave no more result after a first run. Since 2.0 I have three results. But I know there must be much more alignnone in my content. So I tried SearchReplace script from Interconnect and it found more than 300 entries.

#### Searchregex

<img width="1675" alt="regex" src="https://user-images.githubusercontent.com/689870/81581143-a3356700-93ae-11ea-8e52-fe949bb436fe.png">

#### SearchReplace script

<img width="1253" alt="search-replace" src="https://user-images.githubusercontent.com/689870/81581156-a7618480-93ae-11ea-9189-5868a1cb5b00.png">

### Environment

Centerbrowser (Chromium) and WP 5.4.1. All plugins desactivated.

**Comment by johngodley** [comment]
You are searching in the post type 'articles', for which you have 3 rows.

I don't know what SearchReplace is looking in, but it's probably looking in other places as it says it's looking at 5305 rows, which is more than 3.

**Comment by Li-An** [comment]
I made it search in Post table.
Even if the script searches in much more places, I suppose Searchregex should find ALL occurrences and it does not. See https://www.echodesplugins.li-an.fr/plugins/djd-site-post/, https://www.echodesplugins.li-an.fr/plugins/logarithmic-pagination/ and a lot more (some thumbnails are broken, don’t take care, I’m fixing this).

**Comment by johngodley** [comment]
What are those links showing?

As I said, I don't know what SearchReplace is looking at. As such any comparison is like comparing apples and oranges - they are different.

Search Regex looks in the table you specify, and only in certain columns within that table.

Maybe if you can show the SearchReplace results you will see where it is looking for data.

**Comment by Li-An** [comment]
Very strange. A new search with Searchregex gives much more results (the Djd post is included but not the Logarithmic). 

Well, I can understand the two tools do not search and work in the same way but why Searchregex doesn’t find all **alignnone** in my posts content ?

**Comment by johngodley** [comment]
Can you provide an example of one post that Search Regex is not finding content in, and Search Replace is?

At the moment you've shown some numbers, but no specific example that shows exactly where something isn't being found.

**Comment by Li-An** [comment]
OK, it seems more complicated than I thought. I suspect the problem is relative to JP Markdown plugin what manages a second Markdown version of the posts.
I found a post https://www.echodesplugins.li-an.fr/plugins/etikker/ that SearchReplace script shows as result and not Searchregex. When I took a look in edition page, **alignnone** was showing but on frontend, the image was styled **aligncenter**. I saved the post and Searchregex was able to find  the **alignnone**. So I think I will replace all with the script. 

There is a mess somewhere and it’s not relative to your plugin.
